packages/radix-ui-themes/src/components/checkbox-card-group.css

```diff
@@ -0,0 +1,163 @@
+@import './base-checkbox.css';
+
+.rt-CheckboxCardGroupRoot {
+  line-height: var(--line-height);
+  letter-spacing: var(--letter-spacing);
+  user-select: none;
+  cursor: default;
+}
+
+.rt-CheckboxCardGroupItem {
+  min-width: 0;
+
+  display: flex;
+  align-items: center;
+  gap: var(--checkbox-card-group-item-gap);
+  padding: var(--checkbox-card-group-item-padding);
+  border-radius: var(--checkbox-card-group-item-radius);
+
+  /* cursors */
+  cursor: var(--cursor-button);
+  &:where(:has(:disabled)) {
+    cursor: var(--cursor-disabled);
+  }
+}
+
+.rt-CheckboxCardGroupItemInner {
```

Comment:
I've kept the inner for the checkbox rather than going with position absolute because this way it takes part into the layout, thus not reducing the amount of space available for text.

Reply:
Hm, I don't follow how it helps; let's catch up about this